uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
How are matches to multiple watchlist entries handled? Is it just "associate to whichever the system happens to find first" or is there a business rule around match order? I think this is actually a many-to-many relationship..